improver_tests/utilities/test_mathematical_operations.py

```diff
@@ -37,31 +37,31 @@
 import numpy as np
 from iris.tests import IrisTest
 
+from improver.metadata.utilities import generate_mandatory_attributes
+from improver.utilities.cube_manipulation import sort_coord_in_cube
 from improver.utilities.mathematical_operations import Integration
 
-from ..ensemble_calibration.ensemble_calibration.helper_functions import (
-    set_up_temperature_cube)
+from ..set_up_test_cubes import set_up_variable_cube, add_coordinate
 
 
-def set_up_height_cube(height_points, cube=set_up_temperature_cube()):
-    """Create cube with added height dimension. By default the existing
-    set_up_temperature_cube utility is used."""
-    ascending = False
-    if np.all(np.diff(height_points) > 0):
-        ascending = True
+def _set_up_height_cube(height_points, ascending=True):
+    """Create cube of temperatures decreasing with height"""
+    data = 280*np.ones((3, 3, 3), dtype=np.float32)
+    data[1, :] = 278
+    data[2, :] = 276
 
-    cubelist = iris.cube.CubeList([])
-    for height_point in height_points:
-        temp_cube = cube.copy()
-        height_coord = iris.coords.DimCoord(height_point, "height", units="m")
-        if ascending:
-            height_coord.attributes = {"positive": "up"}
-        else:
-            height_coord.attributes = {"positive": "down"}
-        temp_cube.add_aux_coord(height_coord)
-        temp_cube = iris.util.new_axis(temp_cube, "height")
-        cubelist.append(temp_cube)
-    return cubelist.concatenate_cube()
+    cube = set_up_variable_cube(data[0].astype(np.float32))
+    height_points = np.sort(height_points)
+    cube = add_coordinate(
+        cube, height_points, "height", coord_units="m")
```

Comment:
`add_coordinate` declares that `height_points` should be `list`, not `np.ndarray`.

Reply:
Updated docstring to specify `list or numpy.ndarray`